distribution/environments/aws-ec2-ecloud/CloudFormation/cloudformation-template.json

```diff
@@ -0,0 +1,76 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Metadata": {
+    "AWS::CloudFormation::Designer": {
+      "25c92f36-7287-4598-9977-993a6e053ac8": {
+        "size": {
+          "width": 60,
+          "height": 60
+        },
+        "position": {
+          "x": 250.18020629882812,
+          "y": 228.64299392700195
+        },
+        "z": 0,
+        "embeds": []
+      }
+    }
+  },
+  "Parameters" : {
+    "InstanceTypeParameter" : {
+      "Type" : "String",
+      "Default" : "t2.large",
+      "AllowedValues" : ["t2.micro", "m1.small", "m1.large", "t2.large"],
+      "Description" : "Type of EC2 instance you want to run your master in."
+    },
+    "KeyNameParameter": {
+      "Type": "AWS::EC2::KeyPair::KeyName",
+      "Description": "The KeyName that will be used to access this EC2 instance."
+    },
+    "SSHLocation" : {
+      "Description" : "The IP address range that can be used to SSH to the EC2 instance for your Jenkins Essentials instance.",
+      "Type": "String",
+      "MinLength": "9",
+      "MaxLength": "18",
+      "AllowedPattern": "(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})/(\\d{1,2})",
+      "ConstraintDescription": "Must be a valid IP CIDR range of the form x.x.x.x/x"
+    }
+  },
+  "Resources": {
+    "EC2EssentialsInstance": {
+      "Type": "AWS::EC2::Instance",
+      "Properties": {
+        "ImageId": "ami-0c6bb742864ffa3f3",
+        "InstanceType": {"Ref": "InstanceTypeParameter"},
+        "BlockDeviceMappings": [
+          {
+            "DeviceName": "/dev/xvda",
+            "Ebs": {
+              "VolumeSize": "115"
+            }
+          }
+        ],
+         "KeyName" : { "Ref" : "KeyNameParameter" },
+        "SecurityGroups": [
+          { "Ref": "EssentialsMasterSecurityGroup"}
+        ]
+      },
+      "Metadata": {
+        "AWS::CloudFormation::Designer": {
+          "id": "25c92f36-7287-4598-9977-993a6e053ac8"
+        }
+      }
+    },
+    "EssentialsMasterSecurityGroup" : {
+      "Type" : "AWS::EC2::SecurityGroup",
+      "Properties" : {
+        "GroupDescription" : "Enable HTTP access via port 8080 and JNLP port (TODO: restrict agent port)",
+        "SecurityGroupIngress" : [
+          {"IpProtocol" : "tcp", "FromPort" : "22", "ToPort" : "22", "CidrIp" : { "Ref" : "SSHLocation"}},
```

Comment:
For some reason, whatever I do, `SSHLocation` seems like it is ignored, and always `0.0.0.0/0` is used instead:
![image](https://user-images.githubusercontent.com/223853/41899011-a7b62cb4-792b-11e8-98c2-8c65c19009e5.png)

But the command looks fine (if I remove `SSHLocation` param, I get an error that the param is required as expected)

```
aws cloudformation create-stack --stack-name evergreen-test$RANDOM --template-body file:///$PWD/distribution/environments/aws-ec2-ecloud/CloudFormation/cloudformation-template.json --region us-east-1 --parameters ParameterKey=KeyNameParameter,ParameterValue=batmat-[REDACTED]-dev "ParameterKey=SSHLocation,ParameterValue=1.2.3.4/0"
```